tests/sles4sap/trento/deploy_trento.pm

```diff
@@ -25,8 +28,10 @@ sub run {
 
     my $script_run = ' ./';
     # Run the Trento deployment
+
+    record_info('00.040');
```

Comment:
You are using `00.040` in several places below. Perhaps you can assign it in a variable and then when you need to change it, you will only need to change it in only one place.